recipe/meta.yaml

```diff
@@ -6,43 +6,113 @@ package:
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_incremental_microbatch.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_persist_docs.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_query_comment.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_relations.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_show.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_copy.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_snapshot.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_store_test_failures.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_unit_testing.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_utils.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_simple_seed/test_simple_seed.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/adapter/test_hooks/test_hooks.py" %}
+{% set ignore_tests = ignore_tests + " --ignore=tests/functional/materializations/materialized_view_tests/test_materialized_view.py" %}
+
+{% set passed_tests = "" %}
+{% set passed_tests = " tests/functional/test_config.py::TestProfile::test_bad_type" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_from_raw_profiles" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_invalid_credentials" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_missing_target" %}
+{% set passed_tests = passed_tests + " tests/functional/test_config.py::TestProfile::test_missing_type" %}
+{% set passed_tests = passed_tests + " tests/functional/test_connection_manager.py::TestConnectionManagerOpen::test_open" %}
+
 test:
+  source_files:
+    - tests
   imports:
     - dbt.adapters.postgres
   commands:
     - pip check
+    - python -m pytest -vvv tests/unit  [not win]
+    - cmd.exe /C "python -m pytest -vvv tests/unit"  [win]
+    # Only 6 functional tests passed, all other tests in the tests/functional directory needs to build and run docker to start postgres database server.
+    # Error message:
+    #   raise FailedToConnectError(str(e))
+    #   dbt.adapters.exceptions.connection.FailedToConnectError: Database Error
+    #   connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
+    #     Is the server running on that host and accepting TCP/IP connections?
+    # Example on how to start psql server:
+    # https://github.com/dbt-labs/dbt-postgres/blob/v1.9.0/pyproject.toml#L76
+    - python -m pytest -vvv {{ ignore_tests }} {{ passed_tests }}  # [not win]
```

Comment:
If you are passing an explicit list of tests, `{{ passed_tests }}` then the `{{ ignore_tests }}` is redundant, isn't it?

Reply:
In this case readability is lost